lua/entities/gmod_wire_expression2/core/bone.lua

```diff
@@ -36,17 +36,26 @@ function getBone(entity, index)
 	if not bone then
 		bone = entity:GetPhysicsObjectNum(index)
 		entity2bone[entity][index] = bone
+		
+		bone2entity[bone] = entity
+		bone2index[bone] = index
 	end
-	if not bone then return nil end
-	if not bone:IsValid() then return nil end
```

Comment:
Can `GetPhysicsObjectNum` return nil? If so, this change will cause lua errors.

Reply:
It returns nil when requesting a bone index that doesn't exist, but `getBone()`, as far as I'm aware, only gets called in situations where its domain is known to exist. I could insert a check.